scss/styles.scss

```diff
@@ -21,3 +23,86 @@
 .wolox-logo {
   width: 200px;
 }
+
+.main-section {
+  margin: 0 140px;
```

Comment:
I do not recommend giving fixed margins to the sides because these should be different according to the size of the screen. I would use flexbox with `flex-direction: column`, align to the center, put a max-width and margin auto. So we get the same effect.